common/src/java-test/com/zimbra/common/mime/MimeHeaderTest.java

```diff
@@ -235,6 +236,7 @@ public void semiColonSeparated() {
 
     @SuppressWarnings("deprecation")
     @Test
+    @Ignore
```

Comment:
Please raise a Jira ticket or Bugzilla ticket for tests that are being ignored (could group them by file perhaps)
and mention the ticket ID in the Ignore line.